tslint.json

```diff
@@ -10,6 +10,7 @@
         "no-internal-module": true,
         "no-trailing-whitespace": true,
         "no-var-keyword": true,
+        "no-unused-variable": true,
```

Comment:
As discussed in person, we'll put in both and I'll expand my miiiiiiind
![miiiiind](http://i.giphy.com/XAWzKcmQZBjxe.gif)